### PR TITLE
feat(cph): add resource adb command

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:
feat(cph): add resource adb command

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```
feat(cph): add resource adb command
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST="./huaweicloud/services/acceptance/cph" TESTARGS="-run TestAccCphAdbCommand_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cph -v -run TestAccCphAdbCommand_basic -timeout 360m -parallel 4
=== RUN   TestAccCphAdbCommand_basic
=== PAUSE TestAccCphAdbCommand_basic
=== CONT  TestAccCphAdbCommand_basic
--- PASS: TestAccCphAdbCommand_basic (30.49s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cph       30.548s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+			return nil, "ERROR", err
+		}
+
+		getJobsRespBody, err := utils.FlattenResponse(getJobsResp)
+		if err != nil {
+			return nil, "ERROR", err
+		}
+
+		runningStatus := utils.PathSearch("jobs[?status==`1`]", getJobsRespBody, make([]interface{}, 0)).([]interface{})
+		failedPhoneIds := utils.PathSearch("jobs[?status==`-1`].phone_id", getJobsRespBody, make([]interface{}, 0)).([]interface{})
```
Comment (wuzhuanhong) [inline huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go]: It is recommended to add corresponding comments for status value.

File: docs/resources/cph_adb_command.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+* `phone_ids` - (Optional, List, NonUpdatable) Specifies the IDs of the CPH phone.
+
+* `server_ids` - (Optional, List, NonUpdatable) Specifies the IDs of CPH server.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
```
Comment (saf3dfsa) [inline docs/resources/cph_adb_command.md:53]: add timeout block

File: huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+// @API CPH GET /v1/{project_id}/cloud-phone/jobs
+func ResourceAdbCommand() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAdbCommandCreate,
+		UpdateContext: resourceAdbCommandUpdate,
+		ReadContext:   resourceAdbCommandRead,
+		DeleteContext: resourceAdbCommandDelete,
+
+		CustomizeDiff: config.FlexibleForceNew(adbCommandNonUpdatableParams),
+
```
Comment (saf3dfsa) [inline huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go:42]: add default timeouts

File: huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go (hunk 1 above)
Comment (luoping-12345) [reply huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go]: image

File: docs/resources/cph_adb_command.md (hunk 2 above)
Comment (luoping-12345) [reply docs/resources/cph_adb_command.md:53]: image

File: huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go (hunk 3 above)
Comment (luoping-12345) [reply huaweicloud/services/cph/resource_huaweicloud_cph_adb_command.go:42]: image

**Pull request merged**